From git-commits-head-owner@vger.kernel.org  Mon Oct  9 23:39:43 2006
Date: Fri, 22 Sep 2006 22:59:03 GMT
Message-Id: <200609222259.k8MMx3nR026426@hera.kernel.org>
From: Jeff Garzik <jeff@garzik.org>
To: stable@kernel.org
Subject: [netdrvr] mv643xx_eth: fix obvious typo, which caused build breakage

From: Jeff Garzik <jeff@garzik.org>

The last minute fix submitted by the author fixed a bug, but
broke the driver build.

Noticed by Al Viro, since I can't build on said platform.

Signed-off-by: Jeff Garzik <jeff@garzik.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 drivers/net/mv643xx_eth.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- linux-2.6.18.orig/drivers/net/mv643xx_eth.c
+++ linux-2.6.18/drivers/net/mv643xx_eth.c
@@ -385,7 +385,7 @@ static int mv643xx_eth_receive_queue(str
 	struct pkt_info pkt_info;
 
 	while (budget-- > 0 && eth_port_receive(mp, &pkt_info) == ETH_OK) {
-		dma_unmap_single(NULL, pkt_info.buf_ptr, RX_SKB_SIZE,
+		dma_unmap_single(NULL, pkt_info.buf_ptr, ETH_RX_SKB_SIZE,
 							DMA_FROM_DEVICE);
 		mp->rx_desc_count--;
 		received_packets++;
